src/vs/workbench/contrib/scm/browser/repositoryPane.ts

```diff
@@ -639,6 +639,8 @@ export class ToggleViewModeAction extends Action {
 
 export class RepositoryPane extends ViewPane {
 
+	private static commitFontFamily = '-apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif';
```

Comment:
Let's move this into the default setting value contribution in:

https://github.com/microsoft/vscode/blob/ee8a6bc4f8a46cd709111b231d950b4468981676/src/vs/workbench/contrib/scm/browser/scm.contribution.ts#L76